src/app/model/cacke/cacke.ts

```diff
@@ -0,0 +1,10 @@
+import { CackeStatus } from './status';
+
+export interface Cacke  {
```

Comment:
As a good practice, you can add the letter I before the name of your interfaces, to identify them versus the classes. Ex. `IBread` or `iBread`.